Documentation for setting the BASE_URL

nickmoreton opened the pull request:
# Documentation for setting the BASE_URL

This adds the information to the main readme for the usage of the BASE_URL setting

Ticket URL: https://projects.torchbox.com/projects/wordpress-to-wagtail-importer-package/tickets/93

WAGTAIL_WORDPRESS_IMPORTER_BASE_URL is renamed to WAGTAIL_WORDPRESS_IMPORTER_SOURCE_DOMAIN

---

- Testing
    - [x] CI passes
- Documentation.
    - [x] This PR adds or updates documentation

nickmoreton committed:
add documentation for setting the BASE_URL in settings

nimasmi commented on README.md:
```diff
@@ -35,6 +36,14 @@ The package has been developed and tested with:
+
+Set the `BASE_URL` setting in your sites settings to the URL of the website that the images and documents will be imported from.
+
+*If you need to use a different URL in `BASE_URL` you can use `WAGTAIL_WORDPRESS_IMPORTER_BASE_URL`.*
```
**Comment**: I'd say this is the other way round — `WAGTAIL_WORDPRESS_IMPORTER_BASE_URL` is the standard way of setting it, but if not set it defaults to BASE_URL. On the other hand, I'm not sure why it defaults to another setting. Some projects may make use of it, but it's not part of core Django.

1. Would it make more sense to remove the fallback?
1. Would it make more sense to implementers if we renamed this to e.g. "WAGTAIL_WORDPRESS_IMPORTER_SOURCE_DOMAIN" (or something else, if I've misunderstood the role)?

As far as I see, this is only used in test settings overrides, and in one configuration module.

nickmoreton replied:
Thanks Nick, what you are saying does make sense. I will alter the implementation for this and update the docs.

nimasmi commented on README.md:
```diff
@@ -35,6 +36,14 @@ The package has been developed and tested with:
+### BASE_URL for importing images and documents
+
+The importer will need to know what the URL is for the website to download images and documents from.
+
```
**Suggestion**: This says the same as the next paragraph.

```suggestion
```

nickmoreton committed:
Remove BASE_URL requirement

WAGTAIL_WORDPRESS_IMPORTER_BASE_URL is renamed to WAGTAIL_WORDPRESS_IMPORTER_SOURCE_DOMAIN

nimasmi reviewed (approved):
This looks good. Thanks Nick.
